server/api/routes/main.py

```diff
@@ -1,8 +1,20 @@
 from flask import Blueprint
+from database import *
+import json
 
 main = Blueprint('main', __name__)
 
 
 @main.route('/')
 def home():
     return "Hello, world"
+
+
+@main.route('/users')
+def users():
+    out = ""
+    users = db.users.find({})
```

Comment:
This line is causing an error saying that the MongoEngine object has no attribute `users`. From looking at the MongoEngine docs, I changed the line to use `User.objects` and it seemed to get all the objects properly.

Reply:
Whoops, leftover from pymongo